src/main/java/com/sphy/hotelmanagementapplication/domain/Order.java

```diff
@@ -0,0 +1,95 @@
+package com.sphy.hotelmanagementapplication.domain;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long Id;
+
+    private LocalDate checkInDate;
+    private LocalDate checkOutDate;
+
+    @ManyToOne
+    @JoinColumn(name = "client_id")
+    private Client client;
+
+    @ManyToOne
+    @JoinColumn(name = "rooms_id")
+    private Room rooms;
+
```

Comment:
@georgepoul  you also need to add a String property with the name of transactionId, in which we will store the id that is going to be generated from the 3rd party library that we will use for letting the user to be able to use a credit/debit card